src/Nri/Ui/SegmentedControl/V7.elm

```diff
@@ -102,22 +102,25 @@ viewTab maybeToUrl config option =
                         children
     in
     element
-        [ Role.tab
-        , css sharedTabStyles
-        , css <|
-            if option.value == config.selected then
-                focusedTabStyles
-
-            else
-                unFocusedTabStyles
-        , css <|
-            case config.width of
+        (List.concat
+            [ [ Role.tab
+              , css sharedTabStyles
+              ]
+            , if option.value == config.selected then
+                [ css focusedTabStyles
+                , attribute "aria-current" "page"
```

Comment:
🎨 We could use https://package.elm-lang.org/packages/tesk9/accessible-html-with-css/latest/Accessibility-Styled-Aria#currentPage here if we wanted